parsec/test/mocktests/mock_server_test.go

```diff
@@ -53,7 +53,7 @@ func waitForServerSocket() error {
 			// parsec socket exists, exit
 			fmt.Fprintf(ginkgo.GinkgoWriter, "found parsec socket\n")
 			// need short wait till socket can connect
-			time.Sleep(100 * time.Millisecond)
+			time.Sleep(200 * time.Millisecond)
```

Comment:
Could this be replaced with a check of some sort, since it could take quite a different amount of time on different platforms?